tensorflow/examples/ios/caption/CaptionGenerator.h

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
We should use the standard TensorFlow Apache License header here for consistency (even though some of the .m/.mm files don't, we have automated tools to try and keep everything the same elsewhere):

/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
==============================================================================*/